### PR TITLE
Improves the design of messages screen

### PR DESCRIPTION
**Pull request by wrightmartin**
Also adds a new style for manager overview, and adds it to the recipients screen

### CODE REVIEW

File: nuntium/templates/nuntium/profiles/messages_per_instance.html [hunk 1]
```diff
@@ -81,8 +101,19 @@
...
-  <div class="modal-dialog modal-lg">
-    <div class="modal-content b-color">
+  <div class="modal-dialog">
+    <div class="modal-content">
+     <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title">{% trans "Delete this message?"%}</h4>
+      </div>
+      <div class="modal-body">
+        <p>{% trans "Are you sure you want to delete this message? This can't be undone."%}</p>
```
Comment (wrightmartin) [inline nuntium/templates/nuntium/profiles/messages_per_instance.html]: This is all just stubbed out at the moment, needs wiring up #650

File: nuntium/templates/nuntium/profiles/messages_per_instance.html [hunk 2]
```diff
@@ -21,56 +21,76 @@
...
-          <tr>
+          <tr class="message-in-message-list">
             <td>
-              {{ message.subject }}
-              {% if message.public and message.confirmated %}
-              <a target="_blank" href="{% url 'message_detail' slug=message.slug instance_slug=writeitinstance.slug %}"><i class="fa fa-external-link"></i></a>
+              <h3><a href="{% url 'message_detail_private' subdomain=writeitinstance.slug pk=message.pk %}">{{ message.subject }}</a></h3>
+              <p class="message-in-message-list__meta">
+                {% blocktrans with created=message.created|timesince author_name=message.author_name %}
+                  Sent <strong>{{ created }}</strong> ago to <strong>{{ author_name }}</strong> from <strong>{{ author_name }}</strong>
```
Comment (wrightmartin) [inline nuntium/templates/nuntium/profiles/messages_per_instance.html]: author_name in one instance here needs to be 'from' as discussed with @chrismytton

**Comment by wrightmartin** [comment]
screen shot 2015-03-31 at 15 32 18

Relevant screenshot

File: nuntium/templates/nuntium/profiles/messages_per_instance.html (hunk 1 above)
Comment (lfalvarez) [inline nuntium/templates/nuntium/profiles/messages_per_instance.html]: Probably as discussed in #611 we wouldn't need to delete a message just turn it 'private'.

**Comment by chrismytton** [comment]
Fixes https://github.com/ciudadanointeligente/write-it/issues/611

**Comment by chrismytton** [comment]
Fixes https://github.com/ciudadanointeligente/write-it/issues/650

**Pull request merged**